build/README.md

```diff
@@ -1,24 +1,24 @@
-# ODS-CI Container Image
+# RHODA-CI Container Image
 
 A [Dockerfile](Dockerfile) is available for running tests in a container.
-The latest build can be downloaded from: https://quay.io/odsci/ods-ci:latest
+The latest build can be downloaded from: https://quay.io/rhodaci/rhoda-ci:latest
 
-eg: podman pull quay.io/odsci/ods-ci:latest
+eg: podman pull quay.io/rhodaci/rhoda-ci:latest
 
 ```bash
 # Build the container (optional if you dont want to use the latest from quay.io/odsci)
-$ podman build -t ods-ci:master -f build/Dockerfile .
+$ podman build -t rhoda-ci:master -f build/Dockerfile .
 
 # Mount a file volume to provide a test-variables.yml file at runtime
 # Mount a volume to preserve the test run artifacts
-$ podman run --rm -v $PWD/test-variables.yml:/tmp/ods-ci/test-variables.yml:Z -v $PWD/test-output:/tmp/ods-ci/test-output:Z ods-ci:master
+$ podman run --rm -v $PWD/test-variables.yml:/tmp/rhoda-ci/test-variables.yml:Z -v $PWD/test-output:/tmp/rhoda-ci/test-output:Z rhoda-ci:master
 ```
 
-### Running the ods-ci container image in OpenShift
+### Running the rhoda-ci container image in OpenShift
 
-After building the container, you can deploy the container in a pod running on OpenShift. You can use [this](./ods-ci.pod.yaml) PersistentVolumeClaim and Pod definition to deploy the ods-ci container.  NOTE: This example pod attaches a PVC to preserve the test artifacts directory between runs and mounts the test-variables.yml file from a Secret.
+After building the container, you can deploy the container in a pod running on OpenShift. You can use [this](./ods-ci.pod.yaml) PersistentVolumeClaim and Pod definition to deploy the rhoda-ci container.  NOTE: This example pod attaches a PVC to preserve the test artifacts directory between runs and mounts the test-variables.yml file from a Secret.
 
 ```
 # Creates a Secret with test variables that can be mounted in ODS-CI container
```

Comment:
The above line should say:
# Creates a Secret with test variables that can be mounted in RHODA-CI container